lib/http_eventstore/helpers/parse_entries.rb

```diff
@@ -14,12 +14,13 @@ def create_event(entry)
         id = entry['eventNumber']
         event_id = entry['eventId']
         type = entry['eventType']
+        source_event_uri = entry['id']
```

Comment:
It might be misleading that `source_event_uri` is `entry['id']` and `id` is `entry['eventNumber']`